pick.lic

```diff
@@ -759,7 +818,14 @@ class Pick
       end
       return
     end
+    
+    #if its trash leave it in the box and return
+    if @trash_nouns.find { |thing| item =~ /\b#{item}\b/i }
+      echo "trash item identified, left it in the box for trashing: #{item}"
+      return
+    end
 
+    #possible unrecognized item log it & trash manually just in case.
```

Comment:
What are some examples where a trash noun wouldn't match the noun but would the adjectives in the item name, and that be the correct thing to do?

Reply:
Don't know the answer. Any concerns/actions here we need? Reopen comment if so?